packages/core/integration-tests/test/utils.js

```diff
@@ -0,0 +1,283 @@
+const Bundler = require('parcel-bundler');
```

Comment:
You might notice I have the utils duplicated here and in parcel-bundler. I did this mainly because they currently depend on existing in the same test folder where the tests that are using them live. Pulling them out would require refactoring pretty much all of the tests. Also there are still a couple integration tests that I felt were v1 specific that I kept in parcel-bundler, which created a circular dependency when I did try pulling them out. The simplest and safest solution seemed to be a tiny bit of duplication, but let me know if that's a deal breaker. I was thinking that we could pull out the integration utils in the v2 branch where we are going to need to refactor the tests to work against v2 anyway. This will also allow us to refactor in a more granular way as we add features and unskip the tests one by one instead of doing them all in one go and maybe missing something. Moving around and debugging a bunch of tests has not been the most fun task :P